patches/devtools-frontend-src/tsconfig.json.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tsconfig.json b/tsconfig.json
+index 91fa83559d50f5e7749a7e72fedf597b08818b16..b621a0c3184ad69911dc3827dafd8ee821db3bb1 100644
+--- a/tsconfig.json
++++ b/tsconfig.json
+@@ -8,6 +8,7 @@
+     },
+     "checkJs": true,
+     "allowJs": true,
++    "moduleResolution": "node",
```

Comment:
I'm curious why this is needed by us and not Chromium?

Reply:
An excellent question 😁, to which I don't know then answer. Perhaps some of our Node experts could take a look and suggest if there is a better solution.

Reply:
cc @petemill

Reply:
https://github.com/brave/brave-browser/issues/8767